v2/slogger/context.go

```diff
@@ -14,26 +14,36 @@
 
 package slogger
 
+import "sync"
+
 type Context struct {
 	fields map[string]interface{}
+	lock   sync.RWMutex
 }
 
 func NewContext() *Context {
 	c := new(Context)
 	c.fields = make(map[string]interface{}, 0)
+	c.lock = sync.RWMutex{}
```

Comment:
this should be unnecessary as it is the same as the zero value for RWMutex

Reply:
fixed